app/views/brexit_campaign/_header.html.erb

```diff
@@ -0,0 +1,33 @@
+<% content_for :is_full_width_header, true %>
+
+<%= render "govuk_publishing_components/components/inverse_header", { full_width: true } do %>
+  <div class="full-page-width-wrapper">
+    <%= render 'govuk_publishing_components/components/breadcrumbs',
+      breadcrumbs:  [
+        {
+          title: "Home",
+          url: "/"
+        },
+        {
+          title: "Prepare for EU Exit if you live in the UK"
+        }
+      ],
+      collapse_on_mobile: true,
+      inverse: true
+    %>
+
+    <div class="grid-row">
+      <div class="column-two-thirds">
+        <%= render "govuk_publishing_components/components/title", {
+          title: t('campaign.page_title'),
+          inverse: true
+        } %>
+
+        <%= render 'govuk_publishing_components/components/lead_paragraph', {
```

Comment:
This can be removed now as we've decided not to display a description on this page.